ddl/ddl_api.go

```diff
@@ -3327,6 +3318,68 @@ func getAnonymousIndex(t table.Table, colName model.CIStr) model.CIStr {
 	return indexName
 }
 
+func (d *ddl) CreatePrimaryKey(ctx sessionctx.Context, ti ast.Ident, indexName model.CIStr,
+	idxColNames []*ast.IndexColName, indexOption *ast.IndexOption) error {
+	if !config.GetGlobalConfig().AlterPrimaryKey {
+		return ErrUnsupportedModifyPrimaryKey.GenWithStack("unsupported add primary key, alter-primary-key is true")
```

Comment:
```suggestion
		return ErrUnsupportedModifyPrimaryKey.GenWithStack("unsupported add primary key, alter-primary-key is false")
```

Reply:
The error message is werid, how about changing it to "`Unsupported adding primary key when 'alter-primary-key' configuration is false`"?